### ISSUE TITLE
Login OTP (2FA) support

### ISSUE THREAD

**Issue by dalso0418** [IceWhaleTech/CasaOS#1944]
**Is your feature request related to a problem? Please describe.**

**Security settings are required when logging in.
When opening the web of casaOS to the outside, it is difficult to open because there is no authentication system such as 2FA or OIDC.**

**Describe the solution you'd like**

**It would be nice to support 2fa such as OTP when logging in, and furthermore, OIDC.**

**Describe alternatives you've considered**

**Additional context**

Thank you.

**Comment by zoujisong** [comment]
Yes, I really hope this setting can be implemented in the next version.
As far as I know, it's not difficult.

**Comment by CommanderGL** [comment]
yeah it really isn't. I use my own 2fa authentication app written with nuxtjs.
